plugins/thunderbolt/meson.build

```diff
@@ -1,8 +1,15 @@
 cargs = ['-DG_LOG_DOMAIN="FuPluginThunderbolt"']
+tbt_deps = [plugin_deps, gudev]
+
+if get_option('enable-dell')
+  tbt_deps += libsmbios_c
+  tbt_deps += efivar
+endif
 
 shared_module('fu_plugin_thunderbolt',
   sources : [
     'fu-plugin-thunderbolt.c',
+    'fu-dell-common.c',
```

Comment:
Is it right to include this source file if `get_option('enable-dell')` is false? I'm still not super happy with the sprinkling of dell stuff in the other plugins. Could we maybe use `fu_plugin_check_hwid($GUID_FOR_HardwareID_14)` rather than `fu_dell_supported()`?